src/ms-review.tex

```diff
@@ -2011,6 +2002,25 @@ \section{Summary and outlook}
         innovation program under Grant Agreement no. 824064 and from the Spanish Ministry of Universities through the
         Maria Zambrano Talent Attraction Programme, 2021-2023.
 
+        A special acknowledgment has to be given to our first lead developer, Christoph Deil, who
+        started the \gammapy project and set the foundation for its future success.
+
+        A. Donath, R. Terrier, Q. Remy, A. Sinha, C. Nigro, F. Pintore, B. Khélifi, L. Olivera-Nieto, J. E. Ruiz,
+        K. Brügge, M. Linhoff and J.L. Contreras contributed to the writing of the manuscript.
+        
+        A. Donath, B. Khélifi, C. Boisson, C.Deil, C. van Eldik, D. Berge, E. de Ona Wilhelmi, F. Acero, F. Pintore,
+        M. Cardillo, J. Hinton, J.L. Contreras, M. Fuessling, R. Terrier, R. Zanin, R. López-Coto and S. Funk
+        are current or former members of the Gammapy coordination committee and contributed to promotion, coordination
+        and steering of the \gammapy project.
+
+        A. Aguasca-Cabot, P. Bhattacharjee, K. Brügge, J. Buchner, D. Carreto Fidalgo, A. Chen, M. de Bony de Lavergne,
+        A. Donath, J. V. de Miranda Cardoso, C. Deil, L. Giunti, L. Jouvin, B. Khélifi, J. King, J. Lefaucheur,
+        M. Lemoine-Goumard, J.P. Lenain, M. Linhoff, L. Mohrmann, D. Morcuende, C. Nigro, L. Olivera-Nieto,
+        S. Panny, M. Regeard, Q. Remy, J. E. Ruiz, L. Saha, H. Siejkowski, A. Siemiginowska, A. Sinha,
+        B. M. Sipőcz, R. Terrier, T. Unbehaun, T. Vuillaume contributed to the code development of \gammapy.
```

Comment:
How was this list made? It misses some people I guess, eg: Fabio Pintore

Reply:
I compiled it by hand, added Fabio now.